frontend/src/pages/login/login.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import './login.css';
+import SignUp from "./signup";
+import SignIn from "./signin";
+
+export default function Login() {
+  const [signup, setSignup] = useState(false);
+  const [emailError, setEmailError] = useState('');
+  const [passwordError, setPasswordError] = useState('');
+  const [passConfError, setPassConfError] = useState('');
+
+  const fetchUrl = () => signup ? '/signup' : '/signin';
+
+  const sendData = async (e) => {
+    e.preventDefault();
+    const { errors } = await fetch(fetchUrl(), {
+      method: 'POST',
+      body: new URLSearchParams(new FormData(e.target))
+    })
+    .then((res) => {
+      return res.json()
+    })
+
+    setEmailError('');
+    setPasswordError('');
+    setPassConfError('');
+    if (errors) {
```

Comment:
I don't see the need to use the conditional here. You can use `error?.email` to make sure you don't have an error if `error` is `null/undefined`